sandbox/combined-tracks.html.example

```diff
@@ -25,7 +25,7 @@
     <pre>open http://localhost:9999/sandbox/index.html</pre>
   </div>
 
-  <video id="vid1" class="video-js vjs-default-skin" lang="en" controls preload="auto" width="640" height="264" poster="http://vjs.zencdn.net/v/oceans.png">
+  <video id="vid1" class="video-js vjs-default-skin" lang="en" controls preload="auto" width="640" height="264">
```

Comment:
what about swapping to `//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png`?